index.js

```diff
@@ -1,3 +1,199 @@
+    tags: ['html', 'css', 'javascript'],
+    live: '',
+    source: '',
+  },
+  {
+    thumbnail: 'images/card-img4.png',
+    name: 'Multi-Post Stories',
+    details: [
+      'CANOPY',
+      'images/dot.png',
+      'Back End Dev',
+      'images/dot.png',
+      '2015',
+    ],
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea.',
+    tags: ['html', 'css', 'javascript', 'react', 'redux', 'java', 'python'],
+    live: 'https://www.microverse.com',
+    source: 'https://www.github.com',
+  },
+];
+
+// CREATING CARDS FROM ARRAY DATA AND POPULATING DYNAMICALLY
+
+projects.forEach((project) => {
+  const content = ` <div class="card">
+  <a><img class="wallpaper" src=${project.thumbnail} alt="card image 1" /></a>
+  <div class="card-part2">
+    <div class="project-name">
+      <h2>${project.name}</h2>
+    </div>
+    <div class="card-detail">
+      <p class="detail-1">${project.details[0]}</p>
+      <img class="detail-2" src=${project.details[1]} alt="bullet image" />
+      <span class="detail-3">${project.details[2]}</span>
+      <img class="detail-4" src=${project.details[3]} alt="bullet image" />
+      <span class="detail-5">${project.details[4]}</span>
+    </div>
+    <p class="card-description">${`${project.description.substring(
+    0,
+    150,
+  )}.`}</p>
+    <p class="card-description1">${project.description}</p>
+    <div class="card-lower">
+      <div class="card-tags">
+        ${project.tags.map((tag) => `<p>${tag}</p>`).join('')}
+      </div>
+      <button type="submit" class="card-button">See Project</button>
+      <hr/>
+      <div class="card-actions">
+        <a  href=${
+  project.live
+}><button type="submit" class="actions">See Live<img src="images/card-live.png"/></button></a>
+        <a href=${
+  project.source
+}><button type="submit" class="actions">See Source<img src="images/card-github.png"/></button></a>
+      </div>
+    </div>
+  </div>
+</div>`;
```

Comment:
- [ ] **Required:** You have done an amazing job so far, however, there are a few changes I suggest to move to the next milestone. 
- On the windows popup, kindly prevent the page from scrolling, both on mobile and desktop versions.

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/156609251-35d52a14-d459-40a2-9ad3-8f03a4a37957.png) | ![image](https://user-images.githubusercontent.com/27709832/156609392-dcb5c9a6-6cc9-40be-8229-aa738da6cd2b.png)

- Observe that the popup window does not take 100% screen width.
- The image width is positioned across the horizontal width of the popup modal window.